.gitignore

```diff
@@ -25,25 +25,41 @@ main.jsbundle.meta
 main.jsbundle
 HockeySDK-iOS/
 
-# Android/IJ
+# Android/IntelliJ
 #
-*.iml
+build/
 .idea
 .gradle
 local.properties
 zooniverse-key.keystore
 gradle.properties
+*.iml
 
 # node.js
 #
 node_modules/
 npm-debug.log
+yarn-error.log
```

Comment:
Do we need this? I don't believe we are using Yarn.

Reply:
yeah, we can remove this. I think the update script added it.